src/user/UsersSection.tsx

```diff
@@ -53,16 +52,12 @@ export const UsersSection = () => {
         type: "org.keycloak.storage.UserStorageProvider",
       };
 
-      return Promise.all([
-        adminClient.components.find(testParams),
-        adminClient.users.count(),
-      ]);
+      return adminClient.components.find(testParams);
     },
     (response) => {
-      //should *only* list users when no user federation is configured and uses count > 100
-      setListUsers(
-        !((response[0] && response[0].length > 0) || response[1] > 100)
-      );
+      //should *only* list users when no user federation is configured
+      setListUsers(!(response && response.length > 0));
```

Comment:
Looks like the `response` is always defined according to the type, so this condition can possibly be simplified.
```suggestion
      setListUsers(!(response.length > 0));
```
Or
```suggestion
      setListUsers(response.length === 0);
```